Makefile.common

```diff
@@ -10,6 +10,9 @@
 
 -include config/Makefile
 
+PATH:=$(CURDIR)/bin:$(PATH)
+export PATH
+
```

Comment:
Sorry if I am slow here, why do we need to update the `PATH` here?

[Note that I am not saying this is a bad thing to do].

Reply:
We need to update it somewhere, here seems like the right place.

Reply:
What component does need the new path? AFAICS we also set `PATH` in the Sphinx call, so this should not be needed, right?

I am fine with setting PATH globally and in fact this is how the Dune system works, but then we'd need to tweak quite a few items in the makefile. IMHO not worth it.